appcheck/firebase-appcheck-debug/firebase-appcheck-debug.gradle

```diff
@@ -52,7 +52,10 @@ dependencies {
 
     javadocClasspath 'com.google.auto.value:auto-value-annotations:1.6.6'
 
-    testImplementation project(':integ-testing')
+    testImplementation(project(":integ-testing")){
```

Comment:
out of curiosity why `testImplementation` has a dependency in `integ-testing` dep?

Reply:
hmm no idea.. Will create a bug to track this